tests/TrustlyPaymentTest.php

```diff
@@ -0,0 +1,335 @@
+<?php
+
+// phpcs:disable
+use SwedbankPay\Api\Service\Payment\Resource\Collection\PricesCollection;
+use SwedbankPay\Api\Service\Payment\Resource\Collection\Item\PriceItem;
+use SwedbankPay\Api\Service\Trustly\Request\Purchase;
+use SwedbankPay\Api\Service\Trustly\Resource\Request\PaymentPayeeInfo;
+use SwedbankPay\Api\Service\Trustly\Resource\Request\PaymentPrefillInfo;
+use SwedbankPay\Api\Service\Trustly\Resource\Request\PaymentUrl;
+use SwedbankPay\Api\Service\Trustly\Resource\Request\Payment;
+use SwedbankPay\Api\Service\Trustly\Resource\Request\PaymentObject;
+
+use SwedbankPay\Api\Service\Data\ResponseInterface as ResponseServiceInterface;
+use SwedbankPay\Api\Service\Resource\Data\ResponseInterface as ResponseResourceInterface;
+
+use SwedbankPay\Api\Service\Trustly\Transaction\Request\CreateReversal;
+use SwedbankPay\Api\Service\Trustly\Transaction\Resource\Request\TransactionReversal;
+
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Request\TransactionObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\SaleObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\ReversalObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\TransactionObject as TransactionObjectResponse;
+
+use SwedbankPay\Api\Service\Trustly\Transaction\Request\GetSale;
+use SwedbankPay\Api\Service\Trustly\Transaction\Request\GetReversal;
+use SwedbankPay\Api\Service\Trustly\Transaction\Request\GetTransaction;
+
+use SwedbankPay\Api\Service\Trustly\Transaction\Request\GetSales;
+use SwedbankPay\Api\Service\Trustly\Transaction\Request\GetReversals;
+use SwedbankPay\Api\Service\Trustly\Transaction\Request\GetTransactions;
+
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\SalesObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\ReversalsObject;
+use SwedbankPay\Api\Service\Payment\Transaction\Resource\Response\TransactionsObject;
+// phpcs:enable
+
+class TrustlyPaymentTest extends TestCase
```

Comment:
Codacy found an issue: [The class TrustlyPaymentTest has a coupling between objects value of 26. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=6066095)